lua/lsp-status/statusline.lua

```diff
@@ -97,9 +97,39 @@ local function statusline_lsp(bufnr)
   return symbol .. config.indicator_ok .. ' '
 end
 
+local function statusline_errors(icon, bufh)
+  icon = (icon or '✗') .. ' '
```

Comment:
I'd prefer that the fallback icon here come from `config`, e.g. `config.indicator_errors`. Same comment for the other functions.